openmrs/src/main/java/org/motechproject/openmrs/tasks/constants/DisplayNames.java

```diff
@@ -50,7 +50,7 @@
     public static final String VISIT_UUID = "openMRS.visit.uuid";
```

Comment:
this changes to properties are not related with this ticket, please leave in this PR only changes in EventHelper class